forc-plugins/forc-debug/tests/cli_integration.rs

```diff
@@ -30,31 +28,54 @@ fn test_cli() {
 
     run_cmd.arg(format!("http://127.0.0.1:{}/graphql", port));
 
-    let mut cmd = spawn_command(run_cmd, Some(2000)).unwrap();
+    // Increased timeout to account for rustyline initialization
+    let mut cmd = spawn_command(run_cmd, Some(5000)).unwrap();
+
+    // Handle rustyline's escape sequences before the prompt
+    cmd.exp_string("\u{1b}[?2004h").unwrap();
+
+    // Green >> prompt
+    let prompt = "\u{1b}[38;2;4;234;130m>>\u{1b}[0m ";
```

Comment:
nit: could be made a const; i'd imagine it be useful to have a colour list of prompt prefixes

Reply:
could be in future PR.
maybe light-weight macro for prompt_green or something